### DISCUSSION TITLE
Requests and Problems for EcorisMap! （不具合、ご要望）

### DISCUSSION THREAD

**Discussion by tmizu23** [ecorismap/ecorismap#1] [Announcements]
## 👋 Welcome! ようこそ！
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

EcorisMapの不具合やご要望があればメッセージしてください。日本語でも英語でもOKです。

**Comment by mk6673** [comment]
GIS勉強中です。
素人質問で恐縮です。ラインデータを作成した際のレイヤーのエクスポートを試みましたが、
Zipファイルのダウンロードが開始するものの、中身が出力されませんでした（空のzipのみがダウンロードされる）
解決策がありましたらお教えいただけますと幸いです。OSはwindowsPCより、chromeで開いています。

**Reply by tmizu23** [reply]
ご報告ありがとうございます。
バグのようですね。修正してみます。

**Comment by TeppeiEguchi** [comment]
インポートしたgeoJSONファイルがアプリ上で編集できないのですがそういう仕様なのでしょうか。
インポートしたデータを編集したい場合どうすればよいでしょうか。

**Reply by tmizu23** [reply]
レイヤ一覧で編集のチェックボックスを切り替えると編集できるはずですが
いかがでしょうか？

**Reply by TeppeiEguchi** [reply]
レイヤ一覧の編集の列にチェックボックスが表示されません。

**Reply by TeppeiEguchi** [reply]
ちなみに、インポートしているデータはレイヤ一覧のレイヤ名をクリックして、必要なデータを選択後、共有ボタンでエクスポートしたデータです。
エクスポートしたもののうちgeoJSONファイルを再度エコリスマップに取り込むと編集できないようになっています。

**Reply by tmizu23** [reply]
アプリのバグを確認したので、修正します。
近日中に更新したいと思います。
ご連絡ありがとうございます。
